src/private/WRLDMapView.mm

```diff
@@ -1004,6 +1019,65 @@ - (void)notifyPoiSearchCompleted:(const Eegeo::PoiSearch::PoiSearchResults&)resu
     [self.delegate mapView:self poiSearchDidComplete:result.Id poiSearchResponse:poiSearchResponse];
 }
 
+- (void)notifyRoutingQueryCompleted:(const Eegeo::Routes::Webservice::RoutingQueryResponse&)result
```

Comment:
Both the Poi and incoming Mapscene service do a fair bit of converting cpp result object to obj-c, and the routing one's currently bigger than both combined.  Wonder if it's worth moving the conversion of RoutingQueryResponse->WRLDRoutingQueryResponse to a helper method to keep WRLDMapView a little tidier?

Reply:
Conversion moved from WRLDMapView to WRLDRoutingServiceHelpers.
https://github.com/wrld3d/ios-api/pull/23/commits/3af44ade4ceb3c79d85b15590337b71987915462